sections/discussion/ourprocessreflections.tex

```diff
@@ -0,0 +1,78 @@
+\end{figure}
+\noindent
+When we conducted the sprint planning, we would use planning poker to estimate how many story points each task in the proposed sprint backlog should have.
+The persons with the highest and lowest story points would discuss the reasoning behind it.
+The story points were often very different if there were ambiguities, and people understood the tasks differently.
+This gave everyone the same understanding on how the given tasks should be solved and eliminate most ambiguities. 
+\\
+However, in the middle of the project when we started to develop the application and learning \textit{React}, it started to become less systematic.
+This was often due to it being difficult to estimate how long each task would take to solve.
+When we got more familiar with \textit{React} we were able to estimate how much the tasks took, and it got more systematic again.
+\\\\
+We also stopped conducting the retrospective meetings.
+The Scrum Masters stopped enforcing the retrospective meetings so they started fading out.
+These meetings could have helped making the project more structured as we could have discussed the problems that we faced and the reasons why we did not complete our tasks in time.
+
+\subsubsection{Formal reviews}
+The formal reviews was something that we changed in the middle of the semester, so that two people had to review a pull request together.
+Previously people would review a pull request individually.
+Formal reviews were described in \autoref{sec:formal-reviews}.
+The first part of every workday was spent on reviewing pull requests until all pull requests were reviewed.
+\\\\
+This increased the quality of the pull requests, as more errors were caught when two people had to debate the code.
+This was especially important for big or difficult pull requests as it otherwise would be easy to just approve the pull request than to actually try and understand every part of it.
+This is however more time consuming and more difficult to plan.
+If two people are in process of reviewing a pull request another person can be waiting for them to finish if they are assigned a pull request with one of them. 
+Then when a pull request had been reviewed and if changes are requested, then these people had to come back to review the changes.
+To make this more organized a we created a PR spreadsheet to give an better overview of which pull requests you are assigned to, and who are currently requested to do many or few reviews.
+This can be seen on \autoref{fig:formal-reviews}.
+\begin{figure}[H]
+    \centering
+    \includegraphics[width=\linewidth]{figures/formal-reviews.PNG}
+    \caption{Pull request assignment sheet.}
+    \label{fig:formal-reviews}
+\end{figure}
+
+\subsubsection{Working from home}
+Unlike other semesters, we were not assigned a group room this semester.
+This resulted in us working from home most days in the week, because we thought this would be easier and as productive as if we met at the university.
```

Comment:
```suggestion
This resulted in us working from home most days of the week, because we thought this would be easier and as productive as if we met at the university.
```